style.css

```diff
@@ -0,0 +1,66 @@
+:root{
+    --primary-color: #DAA06D;
+}
+
+*{
+    box-sizing: border-box;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+body{
+    margin: 0;
+    padding: 0;
+}
+/* avatar style */
+.first-avatar{
+    width: 100%;
+    max-width: 50px;
+    border-radius: 50%;
+    background-color: var(--primary-color);
+}
+.second-avatar{
+    width: 100%;
+    max-width: 70px;
+    border-radius: 50%;
+    background-color: var(--primary-color);
+}
+.third-avatar{
+    width: 100%;
+    max-width: 100px;
+    border-radius: 50%;
+    background-color: var(--primary-color);
+}
+
+/* badge style */
+.div-flex{
+    display: flex;
+}
+.div-flex p{
+    margin: 0;
+    font-size: 2rem;
+}
+.badge-icon .fa{
+    display: block;
+    width: 100%;
+    font-size: 50px;
+    min-width: 100px;
+    max-width: 100px;
+    border-radius: 50%;
+    background-color: var(#DAA06D);
+}
+.badge-avatar{
+    width: 100%;
+    max-width: 100px;
+    border-radius: 50%;
+    background-color: var(#DAA06D);
+}
+.badge{
+    align-self:flex-end;
+    position: absolute;
+    background-color: var(--primary-color);
+    border-radius: 50%;
+    font-size: 100%;
+    height: 20px;
+    width: 20px;
+    text-align: center;
+    color: whitesmoke;
```

Comment:
Use hex or rgba values of colors and do store it in variables by naming it as --badge-color:whitesmoke in a common variables.css file if needed.